### ISSUE TITLE
inner_products_local does not seem to have a softmax applied, as indicated in the paper.

### ISSUE THREAD

**Issue by kuerbiskopf** [mc-lan/SmooSeg#6]
Hey! In the paper in equation (5) a softmax is applied to both soft label assignments. I can see in modules.py in line 244 that it is done to the teacher. But I don't see it applied to the student. Am I missing something?
Thank you very much.

**Comment by mc-lan** [comment]
`F.cross_entropy` requires the `input` to be raw (unnormalized) logits, as it internally applies the softmax function. Therefore, there is no need to apply softmax beforehand.
